src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker_Patterns.cs

```diff
@@ -83,13 +83,14 @@ public override BoundNode VisitRelationalPattern(BoundRelationalPattern node)
 
         public override BoundNode VisitNegatedPattern(BoundNegatedPattern node)
         {
-            // PROTOTYPE(ngafter): what is the right behavior here?
+            Visit(node.Negated);
```

Comment:
>Visit(node.Negated); [](start = 12, length = 20)

Please add a test for `not null` pattern.

Reply:
Adding pure null tests in various combinations with the new combinators to `NullableReferenceTypesTestsVsPatterns`.

---
In reply to: [381008578](https://github.com/dotnet/roslyn/pull/41503#discussion_r381008578) [](ancestors = 381008578)